docker/1.4.0/py2/Dockerfile.cpu

```diff
@@ -2,12 +2,11 @@ FROM ubuntu:16.04
 
 LABEL maintainer="Amazon AI"
 LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
-LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
 
 ARG PYTHON_VERSION=2.7
 ARG PYTORCH_VERSION=1.4.0
 ARG TORCHVISION_VERSION=0.5.0
-ARG MMS_VERSION=1.0.8
+ARG MMS_VERSION=1.1.0
```

Comment:
right now we are installing mxnet-model-server; this version does not exist on pypi.  This is, however, the latest version for multi-model-server.